packages/mongodb-memory-server-core/src/MongoMemoryServer.ts

```diff
@@ -201,20 +251,37 @@ export default class MongoMemoryServer {
     return uri;
   }
 
+  /**
+   * Get a mongodb-URI for a different DataBase
+   * @param otherDbName Set this to "true" to generate a random DataBase name, otherwise a string to specify a DataBase name
+   */
   async getConnectionString(otherDbName: string | boolean = false): Promise<string> {
+    // should this function be marked deprecated? because it is just a pass-through to getUri
     return this.getUri(otherDbName);
   }
 
+  /**
+   * Get the Port of the currently running Instance
+   * Note: calls "ensureInstance"
+   */
   async getPort(): Promise<number> {
     const { port }: MongoInstanceDataT = await this.ensureInstance();
     return port;
   }
 
+  /**
+   * Get the DB-Path of the currently running Instance
+   * Note: calls "ensureInstance"
+   */
   async getDbPath(): Promise<string> {
     const { dbPath }: MongoInstanceDataT = await this.ensureInstance();
```

Comment:
Do you really want to call `ensureInstace` here, instead of returning `false`/`null` if not running?